app/Services/Cloning/CloneDataService.php

```diff
@@ -80,21 +91,14 @@ public function clone()
         Log::debug("Cloning done! {$this->building->id}");
     }
 
-    public function transformCloneableData(array $cloneableData): array
+    public static function transformCloneableData(array $cloneableData, InputSource $inputSource): array
```

Comment:
`static`? On line 83 you call this in instance form, right?

Reply:
possibilities and inconsistencies are endless! (changed to static for consistency purposes)